Home_Task_13_1.py

```diff
@@ -0,0 +1,121 @@
+class MyCard:
+
+    def __init__(self, card_num, name, surname, expare_month, expare_year, cvv, pin, balance=0):
+        self.set_card_num(card_num)
+        self.set_person(name, surname)
+        self.set_expare_month(expare_month)
```

Comment:
expire month and date should be combined into smth aka expire_date  - single field that reflects real life expire date